SwrveSDK/SwrveUtils.brs

```diff
@@ -187,78 +293,123 @@ function SwrveCopy(obj as Object) as Object
 	return res
 end function
 
-'Util function depnding on resolution, return supported width and height'
-function SWGetSupportedResolution()
+'Util function for checking to see if the file(or folder) exists'
+function SWCheckForFile(path as String) as Boolean
```

Comment:
Component library compatible file logic.

Reply:
Are there likely other changes required for Component library compatibility?  The script paths in the xml for the brs files?  pkg V libpkg

Reply:
@KevBrodigan No, the file names in the XML are compiled in a sandbox and thus do no require any modifications. Also we only need to us `libpkg` if you are reading a file, such as a json config, that was bundled with the component library.

Reply:
👍